sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/autorest.md

```diff
@@ -549,5 +552,7 @@ directive:
   - from: swagger-document
     where: $..[?(@.name=='$orderby')]
     transform: $['x-ms-client-name'] = 'orderBy'
-
+  - from: definitions.json
+    where: $.definitions.RequestReportRecordContract.properties.backendResponseCode
+    transform: $.type = "integer"
```

Comment:
out of curiosity, why not update swagger in the spec repo directly instead of using directive?

Reply:
Resolved